doc/release/1.12.0-notes.rst

```diff
@@ -150,6 +150,13 @@ from the roots of the polynomial. This is useful for higher order polynomials,
 where expansion into polynomial coefficients is inaccurate at machine
 precision.
 
+New ``block`` function for creating blocked arrays
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Add a new ``block`` function to the current stacking functions ``vstack``,
+``hstack``, and ``stack``. It is similar to Matlab's (tm) square bracket
```

Comment:
Probably no point in doing it here, but keeping the one in the docs might be a good idea. It could help people coming to numpy from MATLAB find this function.